interface/patient_file/summary/demographics.php

```diff
@@ -749,38 +749,65 @@ function setMyPatient() {
 if ($GLOBALS['patient_id_category_name']) {
     $idcard_doc_id = get_document_by_catg($pid, $GLOBALS['patient_id_category_name']);
 }
+$res=sqlStatement("SELECT option_id,activity FROM list_options WHERE list_id='patient_file_menu' ");
 
+while ($row = sqlFetchArray($res)) {
+    $patient_file_menu[$row['option_id']]=$row['activity'];
+}
 ?>
 <table cellspacing='0' cellpadding='0' border='0' class="subnav">
   <tr>
       <td class="small" colspan='4'>
+
+          <?php if($patient_file_menu["history"]=="1"){ ?>
           <a href="../history/history.php" onclick='top.restoreSession()'>
             <?php echo htmlspecialchars(xl('History'), ENT_NOQUOTES); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["report"]=="1"){ ?>
             <?php //note that we have temporarily removed report screen from the modal view ?>
           <a href="../report/patient_report.php" onclick='top.restoreSession()'>
             <?php echo htmlspecialchars(xl('Report'), ENT_NOQUOTES); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["documents"]=="1"){ ?>
             <?php //note that we have temporarily removed document screen from the modal view ?>
           <a href="../../../controller.php?document&list&patient_id=<?php echo $pid;?>" onclick='top.restoreSession()'>
             <?php echo htmlspecialchars(xl('Documents'), ENT_NOQUOTES); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["transactions"]=="1"){ ?>
           <a href="../transaction/transactions.php" onclick='top.restoreSession()'>
             <?php echo htmlspecialchars(xl('Transactions'), ENT_NOQUOTES); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["issues"]=="1"){ ?>
           <a href="stats_full.php?active=all" onclick='top.restoreSession()'>
             <?php echo htmlspecialchars(xl('Issues'), ENT_NOQUOTES); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["ledger"]=="1"){ ?>
           <a href="../../reports/pat_ledger.php?form=1&patient_id=<?php echo attr($pid);?>" onclick='top.restoreSession()'>
             <?php echo xlt('Ledger'); ?></a>
           |
+          <?php }?>
+
+          <?php if($patient_file_menu["external_data"]=="1"){ ?>
           <a href="../../reports/external_data.php" onclick='top.restoreSession()'>
             <?php echo xlt('External Data'); ?></a>
-            <?php if ($GLOBALS['fhir_enable']) { ?>
+          <?php }?>
+
+          <?php if($patient_file_menu["publish"]=="1"){ ?>
```

Comment:
This means that if a user wishes to use FHIR (which btw is a potentially dangerous thing to turn on since a user could submit patient data to a public server; ie. we need a global for this), then the global would need to be turned on and it would need to be in the list.

Reply:
I just thought it should be the same, But since you enlightened me that FHIR is problematic
(to be honest I am not familiar with this feature )
I will return it for the global control and make it an exception